### PR TITLE
[Pfc-wd]: Add pfc-wd support for nephos

### PR DESCRIPTION
**Pull request by leoli-nps**
**What I did**
Add pfc-wd support for nephos.
**Why I did it**
We need to support this feature.
**How I verified it**
Test in the nephos lab.
**Details if related**

### CODE REVIEW

**Comment by leoli-nps** [comment]
Hi lguohan, according to your suggestion, I created this PR separately for pfc-wd, please take a moment to deal with it, thanks.
@lguohan @shine4chen

**Comment by wendani** [comment]
Does it pass the pfcwd test in mgmt. repo?

File: orchagent/pfc_detect_nephos.lua [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+                        if time_left <= poll_time then
+                            redis.call('PUBLISH', 'PFC_WD', '["' .. KEYS[i] .. '","storm"]')
+                            is_deadlock = true
+                            time_left = detection_time
+                        else
+                            time_left = time_left - poll_time
+                        end
+                    else
+                        if pfc_wd_action == 'alert' and pfc_wd_status ~= 'operational' then
+                            redis.call('PUBLISH', 'PFC_WD', '["' .. KEYS[i] .. '","restore"]')
```
Comment (wendani) [inline orchagent/pfc_detect_nephos.lua]: We will soon shift the signal channel name from 'PFC_WD' to 'PFC_WD_ACTION' in COUNTERS_DB. Same comment to line 73. https://github.com/Azure/sonic-swss/pull/794/files line 68

**Comment by leoli-nps** [comment]
Hi wendani, thanks for your reminder, I have updated the corresponding channel name from 'PFC_WD' to 'PFC_WD_ACTION'. In addition, regarding 'pfcwd test in mgmt. repo', our mgmt.repo environment does not support pfcwd test temporarily. However, the pfc-wd function tested in mgmt.repo, we have performed corresponding manual tests in our own environment, and have passed.
@wendani

**Pull request merged**